src/Pages/Dashboard/Chart.jsx

```diff
@@ -74,7 +74,7 @@ class Chart extends PureComponent {
               tick={{ fill: "white" }}
               stroke="#efefef"
               type="number"
-              domain={["dataMin", "dataMax"]} //pass data from Dashboard?
+                domain={['dataMin', 'dataMax']}
```

Comment:
Ah, it looks like your code formatter rejects mine. 😄 I'm using [prettier](https://prettier.io/), which prefers double quotes for string constants, and 2 spaced tabs. I'd prefer to be consistent in the project but that's okay, I'll write an eslint configuration for the project so we can have a consistent style.